poc/OwinTestApp/src/OwinClassLibrary/Utilities/MAuthClient.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Net.Http;
+using System.Threading.Tasks;
+using OwinClassLibrary.Authentication;
+using OwinClassLibrary.Cat;
+using OwinClassLibrary.Zipkin;
+
+namespace OwinClassLibrary.Utilities
+{
+    public class MAuthClient : IMAuthClient
+    {
+        private readonly HttpClient _client;
+
+        public MAuthClient(ITraceProvider provider)
+        {
+            _client = new HttpClient(new MAuthMessageHandler(new ZipkinMessageHandler(provider)));
```

Comment:
MAuthClient has dependencies for MAuth and Zipkin (Jordi and Lazlo's suggestion). Maybe it might be better than my suggestion in our situation.